specification/storagemover/resource-manager/readme.md

```diff
@@ -27,18 +27,27 @@ These are the global settings for the storagemover.
 ``` yaml
 openapi-type: arm
 openapi-subtype: providerHub
-tag: package-2023-03
```

Comment:
This should not be removed, we add tags to all versions

Reply:
AFAICT this should be the latest API version, isn't it?

Reply:
Looks like in other services as well it only specifies the latest API version, but I'll also confirm with the PR assignee (closed this PR and opened a new one so the commits generated from the OpenAPI Hub will be clearer https://github.com/Azure/azure-rest-api-specs/pull/24390)